frontend/components/NewItem/NewItemForm/index.js

```diff
@@ -46,26 +46,26 @@ const NewItemForm = withTheme((props) => {
 					/>
 					<View
 						style={{
-							...styles.item.editors,
+							...styles.itemEditors,
```

Comment:
Hmm... pretty sure we shouldn't be changing this. Looks like the rebase may have not collected the most recent changes from master? Double check that you have the most recent version of master with `git pull origin master` and then interactive rebase your branch. If that doesn't fix this, then you may have to manually change this.